ow_faults/src/FaultInjector.cpp

```diff
@@ -90,6 +95,21 @@ void FaultInjector::setComponentFaultsMessage(fault_msg& msg, ComponentFaults va
   msg.value = static_cast<uint>(value);
 }
 
+void FaultInjector::cameraTriggerCb(const std_msgs::Empty& msg){
+  ow_faults::CamFaults camera_faults_msg;
+
+  if (m_faults.camera_left_trigger_failure) {// if fault
+    setComponentFaultsMessage(camera_faults_msg, ComponentFaults::Hardware);
+    m_system_faults_bitset |= isCamExecutionError;
+  } else { //no fault
+    std_msgs::Empty msg;
+    m_camera_trigger_remapped_pub.publish(msg);
+    m_system_faults_bitset &= ~isCamExecutionError;
+  }
+  // publishSystemFaultsMessage();
+  m_camera_fault_jpl_msg_pub.publish(camera_faults_msg);
```

Comment:
This is probably the point where we need to discuss, should we publish the camera fault message only when an image_trigger is active or we keep sending the message regardless.